packages/text-inputs/src/sass/_module.scss

```diff
@@ -42,7 +42,8 @@
 	border-radius: $AU-border-radius;
 	vertical-align: middle;
 	appearance: none;
-
+	width: 100%;
```

Comment:
```suggestion
```

Reply:
No we need this, if I remove it here then I need to add it in all of the different width classes.